src/types/ButtonColor.type.ts

```diff
@@ -0,0 +1,6 @@
+export type ButtonColor =
+  | 'is-primary'
```

Comment:
It seems there is a syntax error with the union type for `ButtonColor`. The first `|` is not needed and should be removed to correctly define the union type.

Reply:
The first `|` is required by `eslint`.